src/integrations/supabase/client.ts

```
@@ -74,35 +74,165 @@ try {
+    range: function(from: number, to: number) { return this; },
+    limit: function(count: number) { return this; },
+    order: function(column: string, options?: any) { return this; },
+    single: () => Promise.resolve({
+      data: null,
+      error: new Error('Supabase not initialized. Check VITE_SUPABASE_URL and VITE_SUPABASE_ANON_KEY in .env')
+    }),
+    maybeSingle: () => Promise.resolve({
+      data: null,
+      error: new Error('Supabase not initialized. Check VITE_SUPABASE_URL and VITE_SUPABASE_ANON_KEY in .env')
+    }),
+    then: function(resolve: any) {
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)